src/content/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -2366,7 +2366,7 @@ Query the `K8sApiServerSample` event in New Relic Insights to see API Server dat
 
 ### Controller manager data
 
-Query the `K8sControllerManagerSample` event in New Relic Insights to see Controller manager data. For more information, see [Configure control plane monitoring](http://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring):
+Query the `K8sControllerManagerSample` event to see Controller manager data. For more information, see [Configure control plane monitoring](http://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring):
```

Comment:
Nitpick: Delete the http://docs.newrelic.com part of the link to `Configure control plane monitoring`